tests/integration/profiles/gpt_neox_20b.json

```diff
@@ -1,8 +1,6 @@
 {
 	"test_series":"performance",
-	"engine":["deepspeed","huggingface"],
-	"model": "gpt-neox-20b",
-	"dtype": ["fp16","bf16"],
+	"model": ["gpt-neox-20b-fp16-deepspeed", "gpt-neox-20b-fp16-huggingface", "gpt-neox-20b-bf16-deepspeed", "gpt-neox-20b-bf16-huggingface", "gpt-neox-20b-smoothquant"],
```

Comment:
One callout : This will change metric_name. Is this going to affect downstream service that processes this metrics?

Reply:
It shouldn't be an issue, but is there a way that we can do this without affecting the old metrics? IE. make a new entry with engine:"deepspeed", model:gpt-neox-20b-smoothquant-int8 with the associated changes to the serving.properties in the prepare.py, and dtype: fp16 can stay. It will generate a new metric just for the SQ version of this model.

Reply:
The metric_name is associate with `model` (https://github.com/deepjavalibrary/djl-serving/blob/master/tests/integration/profiles/gpt_neox_20b.json#L4). Also dtype in json will get expended and applied to all models. But we don't want to run smoothquant only on bf16. I cannot find a easy change to apply to our use case which preserve old metrics_name.

Reply:
The above would have the downside of needing another profile, and stage in the yaml, so it is a tradeoff that we would have to consider.